### PR TITLE
Update russian chain-middleware

### PR DESCRIPTION
**Pull request by 3t3rn1ty**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Closes #XXXXX

### CODE REVIEW

**Comment by RandellDawson** [comment]
@3t3rn1ty Please validate you have complied with all of the items in the list above and translate everything in the instructions.

**Review by term1nal — APPROVED** [review]
Ok.

**Comment by 3t3rn1ty** [comment]
> @3t3rn1ty Please validate you have complied with all of the items in the list above and translate everything in the instructions.

Its ok!

**Comment by RandellDawson** [comment]
@3t3rn1ty You have not translated everything in the instructions.

File: curriculum/challenges/russian/05-apis-and-microservices/basic-node-and-express/chain-middleware-to-create-a-time-server.russian.md [hunk 1]
```diff
@@ -16,9 +16,9 @@ localeTitle: Промежуточное ПО для создания серве
 Подсказка: тест не пройдет, если вы не подключите промежуточное ПО. Если вы смонтируете функцию где-то еще, тест не пройдёт, даже если результат вывода верный.
 </section>
 
-## Instructions
+## Инструкции
 <section id='instructions'>
-In the route <code>app.get('/now', ...)</code> chain a middleware function and the final handler. In the middleware function you should add the current time to the request object in the <code>req.time</code> key. You can use <code>new Date().toString()</code>. In the handler, respond with a JSON object, taking the structure <code>{time: req.time}</code>.
+В цепочке маршрута  <code>app.get('/now', ...)</code> есть функция промежуточного программного обеспечения и конечный обработчик. В функции промежуточного программного обеспечения необходимо добавить текущее время к объекту запроса в ключе <code>req.time</code>. Вы можете использовать <code>new Date().toString()</code>.В обработчике ответьте объектом JSON, принимая структуру <code>{time: req.time}</code>.
```
Comment (greggubarev) [inline curriculum/challenges/russian/05-apis-and-microservices/basic-node-and-express/chain-middleware-to-create-a-time-server.russian.md]: "принимая структуру" - можно использовать более лаконичный перевод, передающий смысл? В данном виде он немного искажен

**Review by greggubarev — CHANGES_REQUESTED** [review]

**Comment by raisedadead** [comment]
Hi @3t3rn1ty can you address @greggubarev's suggestions?

**Pull request closed without merging**